core/modules/field/tests/field_views.test

```diff
@@ -34,7 +34,7 @@ class FieldViewsTestHelper extends ViewsSqlTest {
    */
   public $instances;
 
-  protected function CreateUser($extra_edit = array()) {
```

Comment:
This change is unrelated to this PR and should probably be removed. (Changing class names is likely to have repercussions, so should probably wait until 2.x)

Reply:
Considering this is a protected method and specific a single field + views test, this is unlikely to be used anywhere else. This method should probably be removed entirely and replaced to use the generic `BackdropWebTestCase::backdropCreateUser()`, but that would have been an even larger change.

Reply:
I agree that unrelated changes shouldn't be included here. Better to discuss separately and minimise the changes in this already large PR.

Reply:
kk, I reverted this method name change.